partcad/src/partcad/project_factory.py

```
@@ -11,7 +11,7 @@
 
 
 class ImportConfiguration:
-    def __init__(self, config_obj={}):
+    def __init__(self, config_obj: dict = {}, parent: p.Project | None = None):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)